### PR TITLE
Pud 503 pact test

### PR DESCRIPTION
**Pull request by mikedunfordEE**

### CODE REVIEW

File: server/clients/manageRecallsApi/getDossier.pact.test.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    try {
+      await getDossier(recallId, accessToken)
+    } catch (exception) {
+      expect(exception.status).toEqual(401)
+    }
+  })
+})
+
+function getDossierRequest(description: string, recallId: string, token: string) {
+  return {
```
Comment (AndyNewton) [inline server/clients/manageRecallsApi/getDossier.pact.test.ts:46]: We have 5 PACT GET authorised functions, with this one - differing only in path.
What do you think of pushing a common base function into a helper class for those?
Maybe as a follow up PR?

File: server/clients/manageRecallsApi/getDossier.pact.test.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    withRequest: {
+      method: 'GET',
+      path: `/recalls/${recallId}/dossier`,
+      headers: { Authorization: `Bearer ${token}` },
+    },
+  }
+}
+
+function getDossierResponse(responseBody, expectedStatus = 200) {
+  return {
```
Comment (AndyNewton) [inline server/clients/manageRecallsApi/getDossier.pact.test.ts:57]: Likewise I think we have 7 or 8 copies of this function now.  Refactor soon?

**Review by AndyNewton — APPROVED** [review]
LGTM - but possible follow up re-factorings to more shared helper code?

File: server/clients/manageRecallsApi/getDossier.pact.test.ts (hunk 2 above)
Comment (mikedunfordEE) [reply server/clients/manageRecallsApi/getDossier.pact.test.ts:57]: Yes both sensible suggestions.  Looks like the pact code in the UI is not receiving the love and attention it should if we have this much duplication!  Everyone is obviously doing as I did and cut and pasting ;-)

I'll tackle these in a separate tidy up PR

**Pull request merged**